scripts/generate-sha256.js

```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env node
-
+const path = require('path');
 const shell = require('shelljs');
 
-// Generate the SHA256 for the .vsix that matches the version in package.json
-
-const packageVersion = JSON.parse(shell.cat('package.json')).version;
-const vsix = shell.ls().filter(file => file.match(`-${packageVersion}.vsix`));
+const cwd = process.cwd();
+const vsixfiles = path.join(cwd, 'extensions');
+const vsixes = shell.ls(vsixfiles);
 
-if (!vsix.length) {
-  shell.error('No VSIX found matching the requested version in package.json');
+if (!vsixes.length) {
+  shell.error(
+    'No VSIX files found matching the requested version in package.json'
+  );
   shell.exit(1);
 }
 
-if (/win32/.test(process.platform)) {
-  shell.exec(`CertUtil -hashfile ${vsix} SHA256`);
-} else {
-  shell.exec(`shasum -a 256 ${vsix}`);
+process.chdir('extensions');
```

Comment:
The genreated SHA256 file should have the format
```
6da4adbf213c5deeadf5029cb37691d486649c192f96a8b3b878ddfc6485d37f  salesforcedx-vscode-apex-56.5.1.vsix
```
by changing to the extensions dir we are able to keep this format the same.